### ISSUE TITLE
validation losses are 0

### ISSUE THREAD

**Issue by petri-98** [WongKinYiu/yolov9#127]
Why do I get these 0 validation losses? I'm providing the validation set. Is there anything I should change in cfg files?
![image](https://github.com/WongKinYiu/yolov9/assets/92453562/db0eb3cf-993e-418b-b39f-9f423fe9ca93)

**Comment by WongKinYiu** [comment]
Currently calculation of val loss is closed when use `train_dual.py`.
https://github.com/WongKinYiu/yolov9/blob/main/val_dual.py#L194

**Comment by petri-98** [comment]
Could you please explain this? I did not get it

**Comment by unron0830** [comment]
> Currently calculation of val loss is closed when use `train_dual.py`. https://github.com/WongKinYiu/yolov9/blob/main/val_dual.py#L194

Why is this feature not enabled in train_dual.py?
